base_decorator.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'nameable'
+class BaseDecorator < Nameable
+  def initialize(nameable)
+    super
```

Comment:
- [ ] As I mentioned in the previous comment, kindly make sure you also update this `super` method to this `super()` as right now when trying to `Capitalize` a **Person Name** it raises an `ArgumentError`